mavis/validate/constants.py

```diff
@@ -76,8 +76,8 @@
     'resolve. The assembly will pre-calculate the number of paths (or putative assemblies) and stop if it is greater '
     'than the given setting.')
 DEFAULTS.add(
-    'assembly_min_uniq', 0.01, cast_type=float_fraction,
-    defn='Minimum percent uniq required to keep separate assembled contigs. If contigs are more similar then the lower scoring contig is dropped')
+    'assembly_min_uniq', 0.10, cast_type=float_fraction,
+    defn='Minimum percent uniq required to keep separate assembled contigs. If contigs are more similar then the lower scoring, then shorted, contig is dropped')
```

Comment:
I'm not sure if that definition makes sense anymore.

Reply:
oh, that is definitely a typo. I will remove that